x/evm/client/cli/tx.go

```diff
@@ -66,59 +60,60 @@ func NewRawTxCmd() *cobra.Command {
 				return err
 			}
 
-			clientCtx, err := client.GetClientTxContext(cmd)
-			if err != nil {
-				return err
-			}
-
-			rsp, err := rpctypes.NewQueryClient(clientCtx).Params(cmd.Context(), &etherminttypes.QueryParamsRequest{})
-			if err != nil {
-				return err
-			}
-
-			tx, err := msg.BuildTx(clientCtx.TxConfig.NewTxBuilder(), rsp.Params.EvmDenom)
-			if err != nil {
-				return err
-			}
-
-			if clientCtx.GenerateOnly {
-				json, err := clientCtx.TxConfig.TxJSONEncoder()(tx)
-				if err != nil {
-					return err
-				}
-
-				return clientCtx.PrintString(fmt.Sprintf("%s\n", json))
-			}
-
-			if !clientCtx.SkipConfirm {
-				out, err := clientCtx.TxConfig.TxJSONEncoder()(tx)
-				if err != nil {
-					return err
-				}
-
-				_, _ = fmt.Fprintf(os.Stderr, "%s\n\n", out)
-
-				buf := bufio.NewReader(os.Stdin)
-				ok, err := input.GetConfirmation("confirm transaction before signing and broadcasting", buf, os.Stderr)
-
-				if err != nil || !ok {
-					_, _ = fmt.Fprintf(os.Stderr, "%s\n", "canceled transaction")
-					return err
-				}
-			}
-
-			txBytes, err := clientCtx.TxConfig.TxEncoder()(tx)
-			if err != nil {
-				return err
-			}
-
-			// broadcast to a Tendermint node
-			res, err := clientCtx.BroadcastTx(txBytes)
+			_, err = client.GetClientTxContext(cmd)
 			if err != nil {
 				return err
 			}
-
-			return clientCtx.PrintProto(res)
+			return nil
+
+			//rsp, err := rpctypes.NewQueryClient(clientCtx).Params(cmd.Context(), &etherminttypes.QueryParamsRequest{})
+			//if err != nil {
+			//	return err
+			//}
+			//
+			//tx, err := msg.BuildTx(clientCtx.TxConfig.NewTxBuilder(), rsp.Params.EvmDenom)
+			//if err != nil {
+			//	return err
+			//}
+			//
+			//if clientCtx.GenerateOnly {
+			//	json, err := clientCtx.TxConfig.TxJSONEncoder()(tx)
+			//	if err != nil {
+			//		return err
+			//	}
+			//
+			//	return clientCtx.PrintString(fmt.Sprintf("%s\n", json))
```

Comment:
Q: @facs95 Why is all this code commented out?

Reply:
good catch forgot to uncommented 🤦